draft-ietf-quic-transport.md

```diff
@@ -541,12 +721,54 @@ packet number.  Once any packet has been acknowledged, subsequent packets can
 use a shorter packet number encoding.
 
 
-## Frames and Frame Types {#frames}
+## Handling Packets from Different Versions {#version-specific}
+
+Between different versions the following things are guaranteed to remain
+constant:
+
+* the location and size of the Flags field in both header forms,
```

Comment:
I would simply refer to this as the first octet, which sorts the both the "location" and "size" aspects.  Maybe:

> New versions of QUIC can be specified as long as the invariants in this section are maintained.
>
> The first octet always indicates the type of packet:
>  * The header form bit (0x80) always indicates a long or short header.
>  * In the short header, the next bit (0x40) indicates whether the Connection ID field is present.
>  * The remaining bits of this octet are interpreted as determined by the protocol version.
>
> Between all protocol versions, the following is guaranteed to be constant:
> * The location, size, and semantics of the Connection ID in both header forms (conditional on the field being present in the short form),
> * The location, size, and semantics of the Version field in the long form header, and
> * The location and size (but not semantics) of the Packet Number field in the long form header.